python/src/aac/context/util.py

```diff
@@ -0,0 +1,34 @@
+"""Common utility functions for the context package."""
+from aac.context.language_error import LanguageError
+
+
+def get_python_module_name(package_name: str) -> str:
+    """Returns a python module name from a given package name, which may contain dashes and underscores."""
+    if not package_name or package_name == "":
+        # we're just using default if no package is declared...which really shouldn't happen in the base AaC stuff
+        return "default"
+    result = package_name.replace(" ", "_").replace("-", "_").lower()
+    for item in result.split("."):  # expect name to be dot notation
+        if not item[0].isalpha():
+            raise LanguageError(f"Invalid name for creation of python module name: '{item}' within '{package_name}', the first character must be a letter")
+        for sub_item in item.split("_"):  # and name elements may contain _
+            if not sub_item.isalnum():  # but things between underscore must be alphanumeric
+                raise LanguageError(f"Invalid AaC package name for creation of python module name: '{package_name}', the name must be alphanumeric")
+    return result
+
+
+def get_python_class_name(name: str) -> str:
+    """Returns a python class name from a given name, which may contain dashes and underscores."""
+    if not name or name == "":
+        raise LanguageError(f"Invalid name for creation of python class name: '{name}', the name must be alphanumeric")
+    if not name[0].isalpha():
+        raise LanguageError(f"Invalid name for creation of python class name: '{name}', the first character must be a letter")
+    result = name.replace("-", " ").replace("_", " ").replace(" ", "")
```

Comment:
Making sure I am following the replace logic here. Remove dashes and inject spaces, remove underscores and inject spaces, then remove spaces?

Would we not just be able to do this? 
```suggestion
    result = name.replace("-", "").replace("_", "")
```

Or is it about covering for if the given name includes spaces as well?

Reply:
It's just creating a name without dashes, underscores, or spaces.